src/containers/ArticleListing/state/reducer.js

```diff
@@ -19,33 +19,33 @@ export const initialState = {
 
 const listReducer = (state = articleListingState, action) => {
   switch (action.type) {
-    case FETCH_ARTICLES:
-      return {
-        ...state,
-        isFetching: true,
-      };
-    case FETCH_ARTICLES_SUCCESS: {
-      return {
-        ...state, articles: action.payload.data, isFetching: false,
-      };
-    }
-    case FETCH_MORE_ARTICLES_SUCCESS: {
-      const results = state.articles.results.map(a => ({ ...a }));
-      if (results) results.push(...action.payload.data.results);
-      return {
-        ...state,
-        articles: { ...action.payload.data, results },
-        isFetching: false,
-      };
-    }
-    case FETCH_ARTICLES_FAILURE:
-      return {
-        ...state,
-        errors: action.payload.errors,
-        isFetching: false,
-      };
-    default:
-      return state;
+  case FETCH_ARTICLES:
+    return {
+      ...state,
+      isFetching: true,
+    };
+  case FETCH_ARTICLES_SUCCESS: {
+    return {
+      ...state, articles: action.payload.data, isFetching: false,
+    };
+  }
+  case FETCH_MORE_ARTICLES_SUCCESS: {
+    const results = state.articles.results.map(a => ({ ...a }));
+    if (results) results.push(...action.payload.data.results);
+    return {
+      ...state,
+      articles: { ...action.payload.data, results },
+      isFetching: false,
+    };
+  }
+  case FETCH_ARTICLES_FAILURE:
+    return {
+      ...state,
+      errors: action.payload.errors,
+      isFetching: false,
+    };
+  default:
+    return state;
```

Comment:
Add test for the default, check if the state is empty first in order to trigger this line

Reply:
Seen